pkg/evaluators/identity.go

```diff
@@ -206,7 +206,10 @@ func (config *IdentityConfig) ResolveExtendedProperties(pipeline auth.AuthPipeli
 }
 
 func (config *IdentityConfig) GetOpenIdConfig() auth.OpenIdConfigStore {
-	return config.JWTAuthentication
+	if config.JWTAuthentication != nil {
+		return config.JWTAuthentication
+	}
+	return nil
```

Comment:
Alternatively can add the following `nil` check

```go
	if j == nil {
		return nil, errors.New(msg_jwtVerifierDoesNotStoreOpenIdConfig)
	}
```

at https://github.com/Kuadrant/authorino/blob/390081929a260f20441c606f1fea1675fbe5adf5/pkg/evaluators/identity/jwt.go#L80-L87